packages/core/src/routes/types.ts

```diff
@@ -2,6 +2,7 @@ import Router from 'koa-router';
 
 import { WithAuthContext } from '@/middleware/koa-auth';
 import { WithI18nContext } from '@/middleware/koa-i18next';
+import { WithUserInfoContext } from '@/middleware/koa-user';
 
 export type AnonymousRouter = Router<unknown, WithI18nContext>;
-export type AuthedRouter = Router<unknown, WithAuthContext<WithI18nContext>>;
+export type AuthedRouter = Router<unknown, WithUserInfoContext<WithAuthContext<WithI18nContext>>>;
```

Comment:
make naming consistent would be better.
Naming `WithUserInfoContext` to `WithUserContext` or naming middleware `koa-user` to `koa-user-info`. I prefer the latter one.